app/controllers/App.scala

```diff
@@ -20,9 +24,13 @@ object App extends Controller with PanDomainAuthActions {
       case None => routes.Assets.versioned(jsFileName).toString
     }
 
-    val clientConfig = ClientConfig(Config().capiUrl, Config().capiKey, TagType.list.map(_.name))
+    Permissions.getPermissionsForUser(req.user.email).map { permissions =>
```

Comment:
Permissions are stored in S3, so downtime should be minimal, they are also cached by the application so only needs to get it successfully once, and if all else fails, falls back to the default, which is in this case... no access.